accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+## SDK Pack
+
+An SDK pack is a unit of packaging for the assets that make up the SDK. Packs have an ID and version and are installed into folders in the SDK directory.
+
+Packs will be installed using CLI tooling or by a native installer such as the Visual Studio installer. It is likely that packs will be distributed via NuGet at some point, so it is recommended that pack authors reserve NuGet IDs for their packs even if other distribution methods are used initially.
+
+SDK packs are used for construction, composition and distribution only, they are not exposed through the user experience.
+
+## SDK Workload
+
+An SDK workload is a semantic name describing a set of one or more SDK packs. When a developer wants to be able to build a particular kind of app, they would expect to install a workload. Workloads should correspond to user scenarios.
+
+Here are some examples of SDK workloads and the packs that they might include:
+
+| Workload ID | Content|
+|--|--|
+| xamarin-android | Framework, runtime, build logic, tools and templates for Xamarin.Android.|
+| xamarin-forms | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects. That could be done by depending on the xamarin-ios, xamarin-android and uwp components, or perhaps instead some kind of lightweight Xamarin.Forms host. |
+| xamarin-forms-android | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects on Android by depending on xamarin-android.|
+| xamarin-android-llvm | The LLVM AOT compiler for Android, omitted by default from the xamarin-android component because it’s large and not all apps use it.<br><br>It may make sense for this to depend on the xamarin-android workload as it’s useless by itself.|
+
+# Versioning
+
+## Manifest Versioning
+
+Manifests follows a similar pattern to SDK versioning: each SDK band has a corresponding manifest band. Each SDK uses only the manifest band corresponding to its own band. Manifest versions however are completely independent of SDK patch levels; an SDK should use the latest available versions of the manifests for its band.
+
+To make this relationship more explicit, the SDK band for a manifest is encoded into the manifest’s ID e.g. `Microsoft.NET.Sdk.Android.SdkManifest-5.0.200`. The “version” of a manifest package can then be then simply a monotonic integer value.
+
+## Pack Versioning
+
+A manifest contains a list of pack IDs and their corresponding versions. A single version is defined for each pack: the most up-to-date version of that pack that is expected to be used with that SDK band.
```

Comment:
> A single version is defined for each pack: the most up-to-date version of that pack that is expected to be used with that SDK band.

Unlikely. What's your plan for supporting the 5.0 and 6.0 SDKs (or 6.0 and 7.0) concurrently, as it relates to the packs that are referenced?

Reply:
this is handled by the aliasing system defined later in the document, added a note to that effect